docs/operating/README.md

```diff
@@ -0,0 +1,12 @@
+# Operating
+
+This section is for anyone managing their own TigerBeetle cluster. While tiger beetles thrive even
+in the harshest conditions, there's certainly a preferred way to handle one!
+
+- [Installing](./installing.md) list all the various way to get the freshest TigerBeetle
+  binary.
+- [Hardware](./hardware.md) specifies the host requirements.
+- [Cluster](./cluster.md) specifies the overall cluster requirements and recommendations.
+- [Deploying](./deploying/) spells out deployment process and its variations.
+- [Upgrading](./upgrading.md) explains how to move to a newer TigerBeetle version without
+  downtime.
```

Comment:
Out of scope for these changes, but what do you think of an "Observability" page which explains TB's log messages (and eventually metrics)?

Reply:
Yeah, we should have `operating/observability/` page!